ext/curl/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+#
```

Comment:
I'm adding a response to a comment on Curl commit so that this can be tracked in a thread here:

> Also a question on [34e0e10](https://github.com/nrfconnect/sdk-nrf/commit/34e0e105d534e4c49f0642e73457cdabb49604b4)
> 
> To me it appears that this commit contains both Nordic code, as well as externally managed code.
> Could this commit be split into two commits.
> 
> * Code that is copied as-is from external source.
> * Code that are created / patched by Nordic

We've flagged with NRF_CURL_INTEGRATION on what we've changed and this was discussed with Robert and Carles. @rlubos / @carlescufi Please comment if we should split Curl commit somehow. And if yes, do we put one commit for all files that we've modified, and another one for the rest of the files.

Reply:
Fair enough, feel free to keep commit as-is.
I wasn't aware this had already been discussed.

Just makes it a bit harder to review this PR, as I don't want to review code that we simply copy from else-where.